### ISSUE TITLE
NPE  while loading translations

### ISSUE THREAD

**Issue by P3ridot** [PikaMug/LocaleLib#28]
Latest change 71eac6d3e857fc2d36dfa99ab24f417cf67262cd (on version <= 3.3 everything works normal) for some reason cause `NullPointerException` (even if `assets/minecraft/lang/en_us.json` actually exists in runtime). I tested it on 1.12.2 and 1.16.5 and same error appears (both on Java 8). 

Also I don't think loading english translations always is necessary - it's only used in `LocaleManager#toServerLocale` which don't appear to be used anywhere in LocaleLib itself

```
java.lang.NullPointerException: inStream parameter is null
        at java.util.Objects.requireNonNull(Objects.java:233) ~[?:?]
        at java.util.Properties.load(Properties.java:407) ~[?:?]
        at net.dzikoysk.funnyguilds.libs.me.pikamug.localelib.LocaleKeys.loadTranslations(LocaleKeys.java:987) ~[?:?]
        at net.dzikoysk.funnyguilds.libs.me.pikamug.localelib.LocaleManager.<init>(LocaleManager.java:99) ~[?:?]
        at net.dzikoysk.funnyguilds.FunnyGuilds.onEnable(FunnyGuilds.java:251) ~[?:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263) ~[patched_1.16.5.jar:git-Paper-794]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:370) ~[patched_1.16.5.jar:git-Paper-794]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:500) ~[patched_1.16.5.jar:git-Paper-794]
        at org.bukkit.craftbukkit.v1_16_R3.CraftServer.enablePlugin(CraftServer.java:518) ~[patched_1.16.5.jar:git-Paper-794]
        at org.bukkit.craftbukkit.v1_16_R3.CraftServer.enablePlugins(CraftServer.java:432) ~[patched_1.16.5.jar:git-Paper-794]
        at net.minecraft.server.v1_16_R3.MinecraftServer.loadWorld(MinecraftServer.java:599) ~[patched_1.16.5.jar:git-Paper-794]
        at net.minecraft.server.v1_16_R3.DedicatedServer.init(DedicatedServer.java:298) ~[patched_1.16.5.jar:git-Paper-794]
        at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:1074) ~[patched_1.16.5.jar:git-Paper-794]
        at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$a$0(MinecraftServer.java:291) ~[patched_1.16.5.jar:git-Paper-794]
        at java.lang.Thread.run(Thread.java:831) [?:?]
```

**Comment by PikaMug** [comment]
Thanks for reporting this. `LocaleManager#toServerLocale` is provided for developers. I'll try to look into this when I can, but going to tag @UltraGameCoder in-case they'd rather.

**Comment by UltraGameCoder** [comment]
Thank you for tagging me @PikaMug. I do be interested in finding the solution to this problem.

So far I was able to find out that the locale isn't inside paper jars.
But rather as seen in the printed exception, it uses `patched_1.16.5.jar:git-Paper-794` and that's where I found the locale:
Paper jar: `paper-1.16.5-794.jar`
Located locale jars: `cache/mojang_1.16.5.jar` & `cache/patched_1.16.5.jar`
Resource location within both jars: `assets/minecraft/lang/en_us.json`

**Comment by UltraGameCoder** [comment]
While looking for a solution. I actually stumbled upon a way to retrieve the Minecraft language locale for any language xD
But for that to work it requires the system to have the Minecraft client installed. But that's something to look into another time.

The problem we're having has to do with the fact that Java's properties class cannot load in all file formats.
Therefor I have decided to read the values from the JSON and LANG file formats into an hashmap by only using what is already provided in terms of libraries.
Besides that, I have also chosen to implement a way of scanning for the minecraft locales within the loaded resources.

I will create a PR shortly after testing my code on multiple minecraft versions.
